Src/Base/AMReX_MFIter.H

```diff
@@ -174,7 +175,7 @@ public:
 
     const DistributionMapping& DistributionMap () const noexcept { return fabArray.DistributionMap(); }
 
-    static int allowMultipleMFIters (int allow);
+    static AMREX_EXPORT int allowMultipleMFIters (int allow);
```

Comment:
This is a function.  Does it need `AMREX_EXPORT`?

Reply:
Thanks, that's a mistake. It's already exposed, this would be double.